README.md

```diff
@@ -137,6 +137,16 @@ See [example_test.go](./example_test.go) for more examples.
 Untagged COSE_Sign1 messages can be signed and verified as above, using
 `cose.UntaggedSign1Message` instead of `cose.Sign1Message`.
 
+#### Signing and Verification of digested payloads
+
+When `cose.NewSigner` is used with PS{256,384,512} or ES{256,384,512}, the returned signer
+can be casted to the `cose.DigestSigner` interface, whose `SignDigest` method signs an
+already digested message.
+
+When `cose.NewVerifier` is used with PS{256,384,512} or ES{256,384,512}, the returned verifier
+can be casted to the `cose.DigestVerifier` interface, whose `VerifyDigest` method verifies an
+already digested message.
```

Comment:
Please add a similar construct here as well exemplifying, as you have added above, i.e.

See [example_test.go](https://github.com/veraison/go-cose/blob/82a1648f4a359716c9ab21981c884d780893e9ba/example_test.go) for more examples.

Reply:
Done.

Reply:
But only added an example to example_test.go. This being an advanced feature, I prefer not to pollute the README.md.

Reply:
I never said, we should pollute the README. I just said point the reader to examples as shown below, how to use the interface.

Reply:
Oh, I misunderstood your comment. Done!